src/components/settings/MonitorHistory.vue

```diff
@@ -32,7 +32,14 @@
                 <button class="btn btn-outline-info me-2" @click="shrinkDatabase">
                     {{ $t("Shrink Database") }} ({{ databaseSizeDisplay }})
                 </button>
-                <div class="form-text mt-2 mb-4 ms-2">{{ $t("shrinkDatabaseDescription") }}</div>
+                <i18n-t tag="div" keypath="shrinkDatabaseDescriptionNew" class="form-text mt-2 mb-4 ms-2">
```

Comment:
```suggestion
                <i18n-t tag="div" keypath="shrinkDatabaseDescription" class="form-text mt-2 mb-4 ms-2">
```

Reply:
`shrinkDatabaseDescription` is the old i18n key. I thought you wanted a completely new key:

> A better approach would be to introduce a translation string and remove the current one.

Reply:
given that the other keys are all removed in this version, there is no need to switch to a different key.

Reply:
ok, I moved key to previous place

Reply:
Thanks, that is something that I can't do just by code-reviews ^^